packages/mermaid/src/diagrams/gantt/ganttDb.js

```diff
@@ -100,6 +106,21 @@ export const enableInclusiveEndDates = function () {
   inclusiveEndDates = true;
 };
 
+export const setDateRange = function (txt) {
+  dateRange = txt;
+
+  if (dateRange) {
```

Comment:
Use early return wherever possible. 

```suggestion
export const setDateRange = function (txt) {
  dateRange = txt;

  if (!dateRange) {
     return;
  }
```

Reply:
That's the opposite convention of the code base I'm working in lately, but happy to go with the house style. :laughing: